packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/getTerraFormFactor.js

```diff
@@ -0,0 +1,23 @@
+import terraViewports from '../../../utils/viewports';
+
+/**
+ * Determines the Terra form factor to for the current viewport size.
+ * @param {Number} browserWidth - current width of the viewpoint
+ * @returns {String} Terra form factor the current viewport with falls under
+ */
+// eslint-disable-next-line
```

Comment:
What rules are failing here?

Reply:
nothing actually, it was junk from copying and pasting. Removed